estate/views/estate_property_offer_views.xml

```diff
@@ -0,0 +1,25 @@
+<odoo>
+    <record id="estate_property_offer_action" model="ir.actions.act_window">
+        <field name="name">Estate Property Offer</field>
+        <field name="res_model">estate.property.offer</field>
+        <field name="view_mode">tree,form</field>
+    </record>
+
+    <record id="estate_property_offer_view_tree" model="ir.ui.view">
+        <field name="name">estate.property.offer</field>
+        <field name="model">estate.property.offer</field>
+        <field name="arch" type="xml">
+            <tree string="Estate Property Offer">
+                <field name="name"/>
+                <field name="partner_id"/>
+                <field name="validity"/>
+                <field name="price"/>
+                <field name="date_deadline"/>
+                <field name="status"/>
+                <!-- <field name="property_type_id" invisible="1"/> <!- - make the field invisible, it do not need show to the user. leave it as learning stuff.  -->
```

Comment:
some time it is necessary to have fields invisible just for the purpose of using them in other fields attributes, see https://www.odoo.com/documentation/17.0/developer/reference/user_interface/view_architecture.html#python-expression